### DISCUSSION TITLE
Passing responsive styles in array syntax causes errors in console from emotion.

### DISCUSSION THREAD

**Discussion by pacxiu** [chakra-ui/chakra-ui#9211] [General] [answered]
Just repinging for visibility, but I can't get my issue being reopened.

https://github.com/chakra-ui/chakra-ui/issues/9092

:+1:

**Comment by pacxiu** [comment]
@segunadebayo any update?

**Reply by segunadebayo** [reply]
I still can't reproduce this. Can you share a StackBlitz reproduction?

**Reply by pacxiu** [reply]
> I still can't reproduce this. Can you share a StackBlitz reproduction?

There is reproduction updated in the issue itself. Will copy link:

[https://stackblitz.com/edit/stackblitz-starters-g6ihco?file=src%2Fpages%2Findex.tsx](https://stackblitz.com/edit/stackblitz-starters-g6ihco?file=src%2Fpages%2Findex.tsx)

**Reply by segunadebayo** [reply, marked as answer]
I see the issue how. You created a new system without defining any breakpoints

![CleanShot 2024-12-07 at 11 35 42@2x](https://github.com/user-attachments/assets/01c735da-13df-42b2-adfb-c8deeaae45cb)

The fix is to merge your new theme with the existing one by doing:

```ts
const system = createSystem(defaultConfig, config);
```

https://stackblitz.com/edit/stackblitz-starters-ypk81jia?file=src%2Fpages%2Findex.tsx,src%2Fpages%2F_app.tsx
